src/table/utils/handle-key-press.ts

```diff
@@ -301,6 +307,9 @@ export const handleBodyKeyDown = ({
     case 'Tab':
       focusSelectionToolbar(evt.target as HTMLElement, keyboard, evt.shiftKey);
       break;
+    case 'c':
+      evt.ctrlKey && evt.key === 'c' && handleKeyToCopyCell(cell);
```

Comment:
should only need to check for c in the case, `evt.key === 'c'` shouldn't be needed.

Reply:
also we need meta key here as well, that is cmd for mac